.eslintrc

```
@@ -0,0 +1,89 @@
+{
+  "extends": ["plugin:@typescript-eslint/recommended", "prettier", "plugin:prettier/recommended"],
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaVersion": 2018
+  },
+  "plugins": ["jest"],
+  "rules": {
+    "array-bracket-newline": ["error", { "multiline": true }],
```

I had a conflict with prettier with this setup earlier in one of my project. `"multiline": true` is default so I omitted.

```suggestion
    "array-bracket-newline": ["error", "consistent"],
```